components/post_markdown/system_message_helpers.jsx

```diff
@@ -16,13 +16,15 @@ import PostAddChannelMember from 'components/post_view/post_add_channel_member';
 function renderUsername(value, options) {
     return renderFormattedText(value, {...options, markdown: false});
 }
-
+function renderUsernameForUserId(value, options) {
+    return renderUsername(Utils.displayUsername(value, options));
+}
```

Comment:
Can we get some newlines around this function? (I'm surprised this passes our style check.)